op-node/node/node.go

```diff
@@ -402,7 +402,8 @@ func (n *OpNode) initL2(ctx context.Context, cfg *Config, snapshotLog log.Logger
 	} else {
 		n.safeDB = safedb.Disabled
 	}
-	n.l2Driver = driver.NewDriver(&cfg.Driver, &cfg.Rollup, n.l2Source, n.l1Source, n.beacon, n, n, n.log, snapshotLog, n.metrics, cfg.ConfigPersistence, n.safeDB, &cfg.Sync, sequencerConductor, plasmaDA)
+	dacClient := cfg.DACConfig.Client()
```

Comment:
It seems most of the components such as l1source, l2source managed by `OpNode`, what about making dacClient also a field in `OpNode`?

Reply:
I don't think it's necessary, it's handled similar to plasmaDA in code.

Reply:
Ok.